lib/src/models/user_model.dart

```diff
@@ -0,0 +1,19 @@
+class UserModel {
+  const UserModel({
+    required this.fullName,
+    required this.email,
+    required this.password,
+  });
+
+  final String fullName;
+  final String email;
+  final String password;
+
+  factory UserModel.fromJson(Map<String, dynamic> json) {
+    return UserModel(
+      fullName: json['fullName'] as String,
+      email: json['email'] as String,
+      password: json['answers'] as String,
```

Comment:
can we have null on these fields?

Reply:
User cant passdown null values, because their types are 'String' and also 'required' keywords.